composer.json

```diff
@@ -21,8 +21,8 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "composer-plugin-api": "^2.2",
-        "php-tuf/php-tuf": "dev-main",
+        "composer-plugin-api": "^2.6",
+        "php-tuf/php-tuf": "dev-main#cc5ce0c40caf2e909c6fe13b38784fb91fac612a",
```

Comment:
This is the last version of the package the current package is compatible.  If we don't have time to keep them in sync we should just pin to the last commit we are compatible with.